development.md

```diff
@@ -0,0 +1,91 @@
+# Swift Driver Development Guide
+
+## Index
+* [Things to install](#things-to-install) 
+* [The code](#the-code)
+* [Building](#building)
+* [Running tests](#running-tests)
+* [Writing and Generating Documentation](#writing-and-generating-documentation)
+* [Linting](#linting)
+* [Workflow](#workflow)
+* [Resources](#resources)
+
+## Things to install
+* [swiftenv](https://swiftenv.fuller.li/en/latest/installation.html): a command-line tool that allows easy installation of and switching between versions of Swift.
+	* Use this to install Swift 4.0 if you don’t have it already.
```

Comment:
Just an observation, but this file has some inconsistent apostrophes (`’` and `'`). May be preferable to stick to the basic ASCII character for a plain text file like Markdown.

Reply:
yeah, agreed, I think this was from copy-pasting. converted all to `'`